### PR TITLE
feat: add details on grype usage in Docker / working grype-db build in Docker

### PR DESCRIPTION
**Pull request by qdzlug**
### Proposed changes
This PR adds additional detail to the existing Grype walkthrough, along with introducing a new 
tool (Dockerfile) that enables a user to build a docker image with a custom grype database suitable
for use in offline or airgapped use cases.

This PR also contains a minor template fix for the repo.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [X] I have written my commit messages in the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
- [X] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [X] I have added tests (when possible) that prove my fix is effective or that my feature works
- [X] I have checked that all unit tests pass after adding my changes
- [X] I have updated necessary documentation
- [X] I have rebased my branch onto master
- [X] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md [hunk 1]
```diff
@@ -33,66 +33,175 @@ docker buildx imagetools inspect <namespace>/<image>:<version> --format "{{ json
...
-        input: keep
-        results: keep
-        retry_count: 3
-        retry_delay: 10
-
+$ brew install grype`
+````
+**Linux / Windows (WSL2)**
+```shell
+$ curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin`
```
Comment (richardkiene) [inline Walkthroughs/CVEScansWithBuildxAndGrype/README.md:52]: danging '`'

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md [hunk 2]
```diff
@@ -33,66 +33,175 @@ docker buildx imagetools inspect <namespace>/<image>:<version> --format "{{ json
...
+ ```
+* Step 2: Install [grype-db](https://github.com/anchore/grype-db)
+```shell
+$ curl -sSfL https://raw.githubusercontent.com/anchore/grype-db/main/install.sh | sh -s -- -b /usr/local/bin`
```
Comment (richardkiene) [inline Walkthroughs/CVEScansWithBuildxAndGrype/README.md:56]: same here

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md [hunk 3]
```diff
@@ -33,66 +33,175 @@ docker buildx imagetools inspect <namespace>/<image>:<version> --format "{{ json
...
-      existing_input: keep
-      existing_results: delete-before-write
-      on_error:
-        action: fail
-        input: keep
-        results: keep
-        retry_count: 3
-        retry_delay: 10
-
+$ brew install grype`
```
Comment (richardkiene) [inline Walkthroughs/CVEScansWithBuildxAndGrype/README.md:48]: oh yeah and here

**Review by richardkiene — CHANGES_REQUESTED** [review]

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md (hunk 1 above)
Comment (qdzlug) [reply Walkthroughs/CVEScansWithBuildxAndGrype/README.md]: Fixed.

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md (hunk 3 above)
Comment (qdzlug) [reply Walkthroughs/CVEScansWithBuildxAndGrype/README.md]: Fixed.

File: Walkthroughs/CVEScansWithBuildxAndGrype/README.md (hunk 2 above)
Comment (qdzlug) [reply Walkthroughs/CVEScansWithBuildxAndGrype/README.md]: fixed

**Review by richardkiene — APPROVED** [review]

**Pull request merged**